src/test/exporter/test_exporter.cc

```diff
@@ -0,0 +1,662 @@
+#include "gtest/gtest.h"
+#include "exporter/util.h"
+
+#include <string>
```

Comment:
`<vector>` include appears to have been lost in the move ;)  Strictly speaking, also need to include `<utility>` here for `std::pair`.